devtools/maven/src/main/java/io/quarkus/maven/CreateProjectMojo.java

```diff
@@ -77,8 +77,8 @@
     @Parameter(property = "codestartsEnabled", defaultValue = "false")
     private Boolean codestartsEnabled;
 
-    @Parameter(property = "withExampleCode", defaultValue = "true")
-    private Boolean withExampleCode;
+    @Parameter(property = "withoutExampleCode", defaultValue = "false")
+    private Boolean withoutExampleCode;
```

Comment:
Why not use the primitive boolean here instead?

Reply:
If default value is false, we could..